frontend/src/components/DesktopApps/Taskbar.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { IGameData } from "../../../../shared/types";
+import { useDispatch, useSelector } from "react-redux";
+import { IRefPhaserGame, PhaserGame } from "../../game/PhaserGame";
+import UpgradesContainer from "../UpgradesContainer";
+import { sellData } from "../../slices/gameDataSlice";
+import AutoCollector from "../AutoCollector";
+
+interface TaskbarProps {
+  // Add any props you need here

```

Comment:
A lot of this stuff can be cleaned up - remove any `import` statements that are not used, etc